modules/nextflow/src/main/groovy/nextflow/script/ProcessConfig.groovy

```diff
@@ -813,6 +813,13 @@ class ProcessConfig implements Map<String,Object>, Cloneable {
      *      The {@link ProcessConfig} instance itself.
      */
     ProcessConfig errorStrategy( strategy ) {
+        listOfStrategies = ['terminate', 'finish', 'ignore', 'retry']
```

Comment:
Instead of creating a new list for each invocation, it could be used the [ErrorStrategy](https://github.com/nextflow-io/nextflow/blob/34a27733961adbc66635e1a05c3fab8e3e468295/modules/nextflow/src/main/groovy/nextflow/processor/ErrorStrategy.groovy#L25-L25) enum e.g. 

```
strategy.toUpperCase() !in ErrorStrategy.values()
```

maybe even better to declare constant at the top of class, something like this 

```
private static final List<String> VALID_ERROR_STRATEGIES = ErrorStrategy.values()
```